nautobot_golden_config/utilities/helper.py

```diff
@@ -43,7 +43,7 @@ def get_job_filter(data=None):
     elif data.get("device"):
         query.update({"id": data["device"].values_list("pk", flat=True)})
 
-    base_qs = models.GoldenConfigSetting.objects.first().get_queryset()
+    base_qs = Device.objects.none().union(*[x.get_queryset() for x in models.GoldenConfigSetting.objects.all()])
```

Comment:
this is a really "advanced" query. Wha is `none()` achieving here?

Reply:
Changed this to `|` approach.